src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/StartupObjectsEnumProvider.cs

```diff
@@ -47,18 +52,15 @@ internal class StartupObjectsEnumGenerator : IDynamicEnumValuesGenerator
         private readonly Workspace _workspace;
         private readonly UnconfiguredProject _unconfiguredProject;
         private readonly bool _includeEmptyValue;
-
-        /// <summary>
-        /// When we implement WinForms support, we need to set this for VB WinForms projects
-        /// </summary>
-        private static bool SearchForEntryPointsInFormsOnly => false;
+        private readonly bool _searchForEntryPointsInFormsOnly;
 
         [ImportingConstructor]
```

Comment:
Not directly related to your changes but I don't think this `[ImportingConstuctor]` is needed as this type isn't MEF-exported.